utils/build.ps1

```diff
@@ -1217,6 +1227,8 @@ function Build-CMakeProject {
     # Add additional defines (unless already present)
     $Defines = $Defines.Clone()
 
+    Add-PythonDefines $Defines $Platform
```

Comment:
This doesn't belong here - this is adding it universally, and there is no reason to assume that it will be used by the project (e.g. SQLite doesn't use python).

This is simply introducing unnecessary warnings.